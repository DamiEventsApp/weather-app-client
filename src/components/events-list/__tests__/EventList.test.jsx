import React from 'react';
import { shallow } from 'enzyme';
import EventList from '../EventList.component.jsx';
import '../../../../setupTests.js';

describe('EventList', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<EventList />);
        expect(wrapper).toMatchSnapshot();
    });
});