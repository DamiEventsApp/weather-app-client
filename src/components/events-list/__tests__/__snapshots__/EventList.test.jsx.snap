// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventList renders correctly 1`] = `ShallowWrapper {}`;

exports[`EventList renders correctly 2`] = `ShallowWrapper {}`;
